src/screen.cpp

```diff
@@ -971,16 +973,37 @@ static screen_layout_t compute_layout(screen_t *s, size_t screen_width,
     }
 
     // Case 3
+    if (!done) {
+      calculated_width = left_prompt_width + first_command_line_width + autosuggest_total_width;
```

Comment:
The indentation here is off, we use 4 space indents, not tabs.

See also build_tools/style.fish

Reply:
Fixed whitespace in https://github.com/fish-shell/fish-shell/pull/7035/commits/812095b7e60fa27cb052bb4cd9ca4e2ac2b5eca1